kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/RealRenderTester.kt

```diff
@@ -154,7 +154,7 @@ internal class RealRenderTester<PropsT, StateT, OutputT : Any, RenderingT>(
         }
     )
 
-    if (expected.output != null) {
+    if (expected?.output != null) {
```

Comment:
Would be helpful to have a comment here explaining what `expected == null` implies.